app/src/main/java/ar/com/wolox/android/example/TrainingConfiguration.java

```diff
@@ -0,0 +1,11 @@
+package ar.com.wolox.android.example;
+
+public final class TrainingConfiguration {
+
+    private TrainingConfiguration() {
+    }
```

Comment:
If constructor is a method is empty try to put it on a single line. It's a silly thing but it makes code "easier to read"